### PR TITLE
Draft release of 0.10.1

### PR DESCRIPTION
**Pull request by alanmcruickshank**
Draft release of 0.10.1

### CODE REVIEW

**Review by tunetheweb — APPROVED** [review]
LGTM - assuming you'll do a final sync up with latest merges before merging.

**Review by barrywhart — APPROVED** [review]

**Review by tunetheweb — CHANGES_REQUESTED** [review]
This should not be merged until #2652 is merged as we've introduced a bug since 0.10.0 that we need to fix before releasing 0.10.1.

**Review by barrywhart — APPROVED** [review]
Looks good to me -- ready for release!

**Pull request merged**
